integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature

```diff
@@ -48,11 +48,3 @@ Feature: SmartMetering Management - Find Events
       | EndDate              | 2015-09-05T00:00:00.000Z |
     Then 29 mbus events should be returned
       | DeviceIdentification | TEST1024000000001 |
-
-  Scenario: find power failure events from a device
```

Comment:
This seemed like a perfectly logical place to handle the events from the power failure event log in the platform.
It is right between other event logs as described in DSMR 4.
I'll leave this point at the comment here, but this would apply to all code changes replacing power failure event log by outages in GXF.